specs/templates/custom-component.md

```diff
@@ -56,31 +56,37 @@
 
 ### Anatomy 
 
-*Outline the component structure with a diagram of its visual tree (shadow dom). Enumerate key areas of visual customization, such as:*
+*Outline the component structure with a diagram of its visual tree (shadow DOM). Enumerate key areas of visual customization, such as:*
 
 - *Slot Names*
 - *Host Classes*
 - *Slotted Content/Slotted Classes*
 - *CSS Parts*
 
-*Work closely with the visual design partner to co-develop the API and anatomy along side the visual design.*
-
 ### Native form integration
 
+*[Optional for Spright]*
```

Comment:
These are not optional fields in the spec. They should have a plan but the plan may be they are not writing Angular bindings, etc, now. It's up to spec reviewers to say the plan of not having Angular bindings is okay for Spright and not okay for Nimble.